frontend/src/components/job/List.tsx

```diff
@@ -138,7 +145,12 @@ export function JobsListRenderer(props: JobsListRendererProps) {
         {
           id: 'images',
           label: t('Images'),
-          getter: job => {
+          getValue: job =>
+            job
+              .getContainers()
+              .map(c => c.image)
+              .join(''),
```

Comment:
Does this matter that it's `''` instead of `', '`?

Reply:
not really, even if you type `,` it still uses fuzzy search by default